examples/pipelines/simple_pipeline/components/image_filtering/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM --platform=linux/amd64 python:3.8-slim
```

Comment:
Do we need still need to define the `platform` flag here if you're passing it in the `build_image.sh` script at build time